fields/types/cloudinaryimage/CloudinaryImageType.js

```diff
@@ -31,10 +40,17 @@ function getEmptyValue () {
 function cloudinaryimage (list, path, options) {
 	this._underscoreMethods = ['format'];
 	this._fixedSize = 'full';
-	this._properties = ['select', 'selectPrefix', 'autoCleanup', 'publicID', 'folder', 'filenameAsPublicID'];
+	this._properties = ['select', 'selectPrefix', 'autoCleanup', 'publicID', 'folder', 'filenameAsPublicID', 'generateFilename', 'whenExists', 'retryAttempts'];
```

Comment:
It could be clearer, but these are the properties that are serialised for the client. So we don't need to add the filename related ones here. I realised we can also take out the `'publicID', 'folder', 'filenameAsPublicID'` properties as well, the client doesn't use them.